chronos2/dashboard_backend/src/core/configs/root_logger.py

```diff
@@ -3,9 +3,7 @@
 import sys
 from logging.handlers import TimedRotatingFileHandler
 
-from src.core.utils.config_parser import cfg
-
-log_file_path = cfg.files.log_path
+log_file_path = "./src/logs/chronos.log"
```

Comment:
I think this should be configurable too. consider this when improving the configuration.